<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>adminWindow</class>
 <widget class="QMainWindow" name="adminWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>458</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>781</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="helloLable">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="logoutBtn">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(204, 235, 207);</string>
       </property>
       <property name="text">
        <string>Logout</string>
       </property>
       <property name="iconSize">
        <size>
         <width>18</width>
         <height>35</height>
        </size>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>781</width>
      <height>341</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Select Mediction</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="margin">
        <number>12</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="medComboBox">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(235, 242, 236);</string>
       </property>
       <property name="maxVisibleItems">
        <number>10</number>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>25</height>
        </size>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="viewInfoButton">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(204, 235, 207);</string>
       </property>
       <property name="text">
        <string>View Information</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextBrowser" name="textBrowser"/>
     </item>
     <item>
      <widget class="QPushButton" name="exportBtn">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(204, 235, 207);</string>
       </property>
       <property name="text">
        <string>Extract current medication information to PDF file</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEdit_data">
    <property name="title">
     <string>Edit Data</string>
    </property>
    <addaction name="actionCreate_new_Admin"/>
    <addaction name="actionAdd_New_Medication"/>
   </widget>
   <widget class="QMenu" name="menuAccess_Data">
    <property name="title">
     <string>Access Data</string>
    </property>
    <addaction name="actionView_All_Admins_Data"/>
    <addaction name="actionView_All_Medication_Data"/>
    <addaction name="actionView_All_Messages"/>
   </widget>
   <addaction name="menuEdit_data"/>
   <addaction name="menuAccess_Data"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionCreate_new_Admin">
   <property name="text">
    <string>Crete new Admin</string>
   </property>
  </action>
  <action name="actionAdd_New_Medication">
   <property name="text">
    <string>Add New Medication</string>
   </property>
  </action>
  <action name="actionView_All_Admins_Data">
   <property name="text">
    <string>View All Admins Data</string>
   </property>
  </action>
  <action name="actionView_All_Medication_Data">
   <property name="text">
    <string>View All Medication Data</string>
   </property>
  </action>
  <action name="actionView_All_Messages">
   <property name="text">
    <string>View All Messages</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
